src/DataTable/SideFilters.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext, useMemo } from 'react';
+import DataTableContext from './DataTableContext';
+
+/** The first filter will be as an input, additional filters will be available in a dropdown.  */
+const SideFilters = () => {
+  const { columns } = useContext(DataTableContext);
+
+  const availableFilters = useMemo(() => columns.filter((column) => column.canFilter), [columns]);
+
+  return (
+    <div className="pgn__data-table-side-filters" title="Filters">
+      <h4>Filters</h4>
```

Comment:
Looking at the Figma spec, it looks like the "Filters" heading could benefit from some additional bottom margin.